source/run-command/tests/unified/runCursorCommand.yml

```diff
@@ -0,0 +1,488 @@
+              command:
+                getMore: { $$type: long }
+                collection: *collection
+                comment: *comment
+                $db: *db
+                lsid: { id: { $$type: binData } }
+              commandName: getMore
+
+  # Tailable cursor
+  - description: does not close the cursor when receiving an empty batch
+    operations:
+      - name: dropCollection
+        object: *db
+        arguments:
+          collection: &cappedCollection cappedCollection
+      - name: createCollection
+        object: *db
+        arguments:
+          collection: *cappedCollection
+          capped: true
+          size: 4096
+          max: 3
+        saveResultAsEntity: *cappedCollection
+      - name: insertMany
+        object: *cappedCollection
+        arguments:
+          documents:
+            - { _id: 1, x: 11 }
+            - { _id: 2, x: 22 }
+      - name: createRunCursorCommand
+        object: *db
+        arguments:
+          cursorType: tailable
+          commandName: find
+          batchSize: 2
+          command: { find: *cappedCollection, filter: {}, tailable: true }
+        saveResultAsEntity: &cursor cursor
+      - name: iterateOnce
+        object: *cursor
+      - name: iterateOnce
+        object: *cursor
+      - name: iterateOnce
+        object: *cursor
+      - name: close
+        object: *cursor
+    expectEvents:
+      - client: *client
+        eventType: command
+        events:
+          - commandStartedEvent:
+              command:
+                drop: *cappedCollection
+              commandName: drop
+          - commandStartedEvent:
+              command:
+                create: *cappedCollection
+              commandName: create
+          - commandStartedEvent:
+              command:
+                insert: *cappedCollection
+              commandName: insert
+          - commandStartedEvent:
+              command:
+                find: *cappedCollection
+                filter: {}
+                $db: *db
+                lsid: { id: { $$type: binData } }
+              commandName: find
+          - commandStartedEvent:
+              command:
+                getMore: { $$type: long }
+                collection: *cappedCollection
+                $db: *db
+                lsid: { id: { $$type: binData } }
+              commandName: getMore
+          - commandStartedEvent:
+              command:
+                killCursors: *cappedCollection
+                cursors: [{ $$type: long }]
+              commandName: killCursors
+
+  # CSOT
+  - description: errors if timeoutMode is set without timeoutMS
+    operations:
+      - name: runCursorCommand
+        object: *db
+        arguments:
+          commandName: find
+          command: { find: *collection, filter: {} }
+          timeoutMode: cursorLifetime
+        expectError:
+          isClientError: true
+
+  - description: error if timeoutMode is cursor_lifetime and cursorType is tailableAwait
```

Comment:
Where was this requirement mentioned? I don't recall reading it in the spec.

Is this client-side validation the main justification for a `cursorType` option?